### PR TITLE
add delete by multiple indexes functionality

### PR DESCRIPTION
**Pull request by elvern18**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteMultipleIndexCommand.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        isIndexesValid(lastShownList);
+
+        ArrayList<Person> listOfPeople = new ArrayList<Person>();
+        for (Index targetIndex : this.indexes) {
+
+            Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
+            listOfPeople.add(personToDelete);
+        }
+        for (Person personToDelete : listOfPeople) {
+            model.deletePerson(personToDelete);
```
Comment (joellow88) [inline src/main/java/seedu/address/logic/commands/DeleteMultipleIndexCommand.java:40]: As the undoManager saves the state of the addressBook after every deletePerson() method call, this implementation of multiple delete causes undo to restore one person at a time.

Instead, I suggest a new method, `deleteMultiplePersons(List<Person>)` be added to the modelManager, that saves the addressBook state once, after all the deletions have completed.

**Review by joellow88 — CHANGES_REQUESTED** [review]
Thanks for implementing the multiple delete feature! Some changes are required as the implementation breaks compatibility with undo. The required changes have been highlighted as a comment to the code.

Do lmk if you need clarification on how the UndoManager works!

File: src/main/java/seedu/address/logic/commands/DeleteMultipleIndexCommand.java (hunk 1 above)
Comment (elvern18) [reply src/main/java/seedu/address/logic/commands/DeleteMultipleIndexCommand.java]: Have just done so. You can take a look at it if you want. Apologies for overlooking this.

**Review by joellow88 — APPROVED** [review]
Thanks for changing the multiple delete so it works with the undo! LGTM!

**Pull request merged**